### ISSUE TITLE
Installation - Basic Question

### ISSUE THREAD

**Issue by chrismalek** [jplattel/obsidian-query-language#24]
I am a bit confused by the installation.

1.) I installed it via the obsidian app menus
2.) I enabled it.
3.) When I try to create a query i get this.

`Index not build yet... (check back after opening another note!)`

I see in the installation instructions that you have some yarn and node commands.   Are those still required to run even if I had obsidian install the plugin?

**Comment by KennSugiyama** [comment]
I'm confused as well, and when I follow the installation steps of cloning before enabling, I get the following error messages:

```
npm WARN tarball tarball data for obsidian@https://github.com/obsidianmd/obsidian-api/tarball/master (sha512-oE7FLE1ICQNKWBm9swMJPlO/iSYtCpau0ANbHx2RNfVyb2/jJMkqcUCcyVYNjwSb1bEn+uU5lkgzrPsk9wJNYA==) seems to be corrupted. Trying again.
npm ERR! code EINTEGRITY
npm ERR! sha512-oE7FLE1ICQNKWBm9swMJPlO/iSYtCpau0ANbHx2RNfVyb2/jJMkqcUCcyVYNjwSb1bEn+uU5lkgzrPsk9wJNYA== integrity checksum failed when using sha512: wanted sha512-oE7FLE1ICQNKWBm9swMJPlO/iSYtCpau0ANbHx2RNfVyb2/jJMkqcUCcyVYNjwSb1bEn+uU5lkgzrPsk9wJNYA== but got sha512-BhXAzwOHjB6jMiIL6EwRRDnE76HMf+MO6BzTm2HY7fE0Vm64RrPZc6rLaAeVs8k5tDccYwKV9yZYGWxXx1iTiA==. (15733 bytes)
```

**Comment by jplattel** [comment]
@chrismalek try restarting Obsidian, it it still doesn't work I'll look into it :)

**Comment by jplattel** [comment]
@KennSugiyama that's an interesting error.. Looks like the zip downloaded was invalid (broken download?) looks like you're trying a manual install with the steps mentioned in the readme?

**Comment by chrismalek** [comment]
> @chrismalek try restarting Obsidian, it it still doesn't work I'll look into it :)

Yes I did restart obsidian several times and even toggled the plugin.

**Comment by KennSugiyama** [comment]
At first I tried downloading through Obsidian, but, like Chris, It was 
stuck on : “Index not build yet... (check back after opening another 
note!).”  After restarted a couple of times, and even creating a new 
vault with only a few notes, I couldn’t get it to work.

I then decided to uninstall and go through the GitHub.  With this 
approach, I tried cloning and manually downing the zip file, without any 
luck.

Take Care,

Kenn Sugiyama

**Comment by KennSugiyama** [comment]
Update:  Not sure why, but after reinstalling through Obsidian, it is now working!

**Comment by tomalbrecht** [comment]
Well, I do have the same problem. I've installed the plugin trough Obsidian. Stuck on  `"Index not build yet... (check back after opening another note!)"`. Tried to reopen Obsidian - still no luck.

I am not able to install QDL via webpage:
```shell
git clone https://github.com/jplattel/obsidian-query-language.git
cd obsidian-query-language
npm install && npm run build
```
Output:
```shell
npm WARN tarball tarball data for obsidian@https://github.com/obsidianmd/obsidian-api/tarball/master (sha512-oE7FLE1ICQNKWBm9swMJPlO/iSYtCpau0ANbHx2RNfVyb2/jJMkqcUCcyVYNjwSb1bEn+uU5lkgzrPsk9wJNYA==) seems to be corrupted. Trying again.
npm ERR! code EINTEGRITY
npm ERR! sha512-oE7FLE1ICQNKWBm9swMJPlO/iSYtCpau0ANbHx2RNfVyb2/jJMkqcUCcyVYNjwSb1bEn+uU5lkgzrPsk9wJNYA== integrity checksum failed when using sha512: wanted sha512-oE7FLE1ICQNKWBm9swMJPlO/iSYtCpau0ANbHx2RNfVyb2/jJMkqcUCcyVYNjwSb1bEn+uU5lkgzrPsk9wJNYA== but got sha512-BhXAzwOHjB6jMiIL6EwRRDnE76HMf+MO6BzTm2HY7fE0Vm64RrPZc6rLaAeVs8k5tDccYwKV9yZYGWxXx1iTiA==. (15733 bytes)

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tom/.npm/_logs/2021-07-15T14_00_14_438Z-debug.log
```

Using MacOS 10.15.7

**Comment by ClaudiaSmily** [comment]
> Update: Not sure why, but after reinstalling through Obsidian, it is now working!

I've tried to re-install Obsidian. No luck. :(

**Comment by chipschipschips** [comment]
jplattel merged a PR that fixed the issue for me. Give it another try and see if that works for you.

If it still doesn't work, try: enabling developer tools (View -> Toggle Developer Tools for Obsidian on OSX); then restart Obsidian; then look for errors in the console. That's where I found the underlying issue that was breaking it for me.
